.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Ned has suggested that we run tests on PRs for all branches, not just those targeting master: https://edx-internal.slack.com/archives/CD89W7Z7F/p1606849364465500

Reply:
Added check to run on all branches.